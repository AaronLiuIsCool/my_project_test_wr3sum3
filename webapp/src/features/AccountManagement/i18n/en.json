{
    "heading": "账号管理",
    "basicInfo": "基本信息",
    "accountManagement": "账号管理",
    "wechat": "微信",
    "email": "邮箱",
    "password": "密码",
    "bind": "前往绑定",
    "unbind": "解除绑定"
  }