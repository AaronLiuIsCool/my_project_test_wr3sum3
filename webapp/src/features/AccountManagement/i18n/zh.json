{
  "heading": "账号管理",
  "basicInfo": "基本信息",
  "accountManagement": "账号管理",
  "wechat": "微信",
  "email": "邮箱",
  "password": "密码",
  "bind": "前往绑定",
  "unbind": "解除绑定",
  "binded": "已绑定",
  "notBinded": "未绑定",
  "reset": "重置",
  "enterEmailAddress": "请输入您所要绑定的邮箱",
  "save": "保存",
  "avatar": "头像",
  "name": "姓名",
  "updateEmail": "更换邮箱",
  "gender": "性别",
  "identity": "身份",
  "genderOptions": [
    {
      "key": "male",
      "value": "男"
    },
    {
      "key": "female",
      "value": "女"
    }
  ],
  "identityOptions": [
    {
      "key": "male",
      "value": "男"
    },
    {
      "key": "female",
      "value": "女"
    }
  ]
}
