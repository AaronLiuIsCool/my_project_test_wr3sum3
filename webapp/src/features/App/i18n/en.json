{
  "brand": "Tailoryourvc",

  "nav_item_resumehub": "Resume Hub",
  "nav_item_jobmatcher": "Jobs",

  "language_toggle_zh": "中文",
  "language_toggle_en": "English",

  "logout": "Sign out",
  "defaultInitial": "Unknown"
}