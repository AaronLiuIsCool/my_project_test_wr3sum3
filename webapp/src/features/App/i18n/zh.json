{
  "brand": "快刀简历",

  "nav_item_resumehub": "我的简历",
  "nav_item_jobmatcher": "工作搜索",
  "nav_item_jobcollction": "我的岗位",
  "language_toggle_zh": "中文",
  "language_toggle_en": "English",
  "createResume": "创建简历",
  "createNewResume": "创建一份全新简历",
  "chooseLanguage": "请选择简历语言",
  "resumeUpload": "我有简历了，我要上传",
  "resumeUploadSupportType": "支持PDF/Word 文档，文件最大支持20MB",
  "uploadYourResume": "从本地上传",
  "upload": "上传",
  "logout": "登出",
  "defaultInitial": "未知",
  "name_it":"请您为这份新简历命名",
  "name_suggestion":"建议用简历类别，比如 [咨询] [投行] [战投] 或者是 [岗位名称+公司名称] 的形式来来命名以方便查询",
  "confirm":"确认",
  "RPreview": {
    "linkedinLink": "领英链接",
    "githubLink": "Github链接",
    "educationBackground": "教育背景",
    "workExperience": "工作经历",
    "projectExperience": "项目经历",
    "studentWorkAndVolunteer": "志愿服务",
    "certificateAndAward": "技能证书 & 获得荣誉",
    "awards": "获得荣誉",
    "certificate": "技能证书",
    "validForever": "永久有效",
    "expiredAt": "到期",
    "current": "现今",
    "editThemeColor": "编辑主题色",
    "smartTranslation": "智能翻译",
    "translationEN": "English",
    "translationCN": "中文版式",
    "oneClickWholePage": "一键整页",
    "downloadResume": "下载简历",
    "perparingResume": "生成中..",
    "downLoadResume": "下载简历",
    "linkedIn": "领英链接",
    "weblink": "项目集链接"
  }
}
