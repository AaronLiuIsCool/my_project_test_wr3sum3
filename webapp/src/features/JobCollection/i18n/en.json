{
    "heading": "我的岗位",
    "all": "全部",
    "applied": "已经申请",
    "postedDate": "发布日期",
    "jobLocation": "岗位所在地",
    "applyJob": "申请工作"
  }