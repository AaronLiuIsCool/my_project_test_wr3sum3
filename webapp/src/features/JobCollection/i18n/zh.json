{
  "heading": "我的岗位",
  "all": "全部",
  "applied": "已经申请",
  "postedDate": "发布日期",
  "jobLocation": "岗位所在地",
  "applyJob": "申请工作",
  "tailoredJobs": "精修岗位",
  "bookmarkedJobs": "已收藏岗位"
}