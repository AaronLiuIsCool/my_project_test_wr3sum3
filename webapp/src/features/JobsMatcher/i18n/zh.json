{
  "search_query_label": "工作岗位搜索",
  "search_query_placeholder": "输入职位或公司",
  "search_country_label": "国家",
  "search_city_label": "城市",
  "search_button": "搜索",

  "job_list_summary": "找到{0}份和您所填经历匹配的工作",
  "job-details-save": "收藏",
  "job-details-referral": "联系内推",
  "job-details-apply": "申请工作", 
  "job_matching_score": "简历匹配度：还行",
  "job_finishing_touches": "对岗精修",
  "job_details_description": "职位描述"
}