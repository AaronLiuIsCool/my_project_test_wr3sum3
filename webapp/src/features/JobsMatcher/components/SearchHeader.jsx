import React, { useState } from 'react';

import { InputGroup, Button, Form, Row, Col } from 'react-bootstrap';
import FormInputGroup from 'components/InputGroup';

import { useI8n } from 'shell/i18n';

import styles from '../styles/SearchHeader.module.css';

const SearchHeader = ({ onSearch, initial = {} }) => {
    const messages = useI8n();
    const [query, setQuery] = useState(initial.query);
    const [country, setCountry] = useState(initial.country);
    const [city, setCity] = useState(initial.city);

    return (
        <Form className={styles.container}>
            <Row>
                <Col lg="6">
                    <Form.Group className="form_item">
                        <Form.Label htmlFor="searchQuery">{messages["search_query_label"]}</Form.Label>
                        <InputGroup>
                            <Form.Control input="text" id="searchQuery"
                                placeholder={messages["search_query_placeholder"]}
                                value={query} onChange={event => setQuery(event.target.value)} />
                            <InputGroup.Append>
                                <Button className={styles["search-button"]} onClick={() => onSearch(query, country, city)}>
                                    {messages["search_button"]}
                                </Button>
                            </InputGroup.Append>
                        </InputGroup>
                    </Form.Group>
                </Col>

                <Col lg="2">
                    <FormInputGroup label={messages["search_country_label"]} id="searchCountry"
                        placeholder={messages["search_country_label"]}
                        value={country} onChange={event => setCountry(event.target.value)} />
                </Col>

                <Col lg="2">
                    <FormInputGroup label={messages["search_city_label"]} id="searchCity"
                        placeholder={messages["search_city_label"]}
                        value={city} onChange={event => setCity(event.target.value)} />
                </Col>

            </Row>
        </Form>
    );
};

export default SearchHeader;