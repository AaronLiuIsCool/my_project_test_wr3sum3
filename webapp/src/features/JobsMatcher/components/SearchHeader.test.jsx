import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';

import store from 'store';
import { I8nContext } from 'shell/i18n';

import SearchHeader from './SearchHeader';

const SearchHeaderWithProvider = (props) => (
    <Provider store={store}>
        <I8nContext.Provider value={{
            'search_query_placeholder': 'query',
            'search_country_label': 'country',
            'search_city_label': 'city'
        }}>
            <SearchHeader {...props} />
        </I8nContext.Provider>
    </Provider>
);

describe('SearchHeader', () => {
    test('render SearchHeader', async () => {
        const searchHeader = renderer.create(
            <SearchHeaderWithProvider />
        );
        expect(searchHeader.toJSON()).toMatchSnapshot();
    });
});
