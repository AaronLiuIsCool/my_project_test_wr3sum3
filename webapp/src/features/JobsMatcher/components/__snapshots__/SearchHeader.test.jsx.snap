// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchHeader render SearchHeader 1`] = `
<form
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-lg-6"
    >
      <div
        className="form_item form-group"
      >
        <label
          className="form-label"
          htmlFor="searchQuery"
        />
        <div
          className="input-group"
        >
          <input
            className="form-control"
            id="searchQuery"
            input="text"
            onChange={[Function]}
            placeholder="query"
          />
          <div
            className="input-group-append"
          >
            <button
              className="search-button btn btn-primary"
              disabled={false}
              onClick={[Function]}
              type="button"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="col-lg-2"
    >
      <div
        className="form_item form-validation-wrapper form-group"
      >
        <label
          className="form-label"
          htmlFor="searchCountry"
        >
          country
        </label>
        <input
          className="form-control"
          id="searchCountry"
          onChange={[Function]}
          placeholder="country"
          type="text"
        />
      </div>
    </div>
    <div
      className="col-lg-2"
    >
      <div
        className="form_item form-validation-wrapper form-group"
      >
        <label
          className="form-label"
          htmlFor="searchCity"
        >
          city
        </label>
        <input
          className="form-control"
          id="searchCity"
          onChange={[Function]}
          placeholder="city"
          type="text"
        />
      </div>
    </div>
  </div>
</form>
`;
